macros/contexts/contextBoolean.pl

```diff
@@ -0,0 +1,160 @@
+## contextBoolean.pl
+
+sub _contextBoolean_init { context::Boolean::Init() }
+
+package context::Boolean;
+
+sub Init {
+	my $context = $main::context{Boolean} = Parser::Context->getCopy("Numeric");
+	$context->{name} = "Boolean";
+	$context->{value}{Formula} = "context::Boolean::Formula";
+
+	## Disable unwanted operations and functions
+	$context->functions->disable('All');
+	#    $context->parens->undefine('<');
+	$context->operators->undefine('-', '/', ' /', '/ ', '//', '^', '**', 'u+', '!');
+	$context->operators->redefine('><');
+
+	## Re-enable necessary operations
+	$context->operators->set(
+		'+'   => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
```

Comment:
You could add to these hashes things like `alternatives  => ["\x{2228}"]` (see https://www.compart.com/en/unicode/U+2228) and then if `$pg{specialPGEnvironmentVars}{parseAlternatives} = 1` is set in localOverrides, students can enter like `x ∨ y` if they copy-paste that from somewhere.

Maybe `v` (the actual letter of the alphabet) should be an alternative too, for this one?

Reply:
There's probably a good number of alternative characters that could be used here... you're right that we should include some of the more obvious ones by default.